examples/pytorch_lightning/aim.py

```diff
@@ -0,0 +1,103 @@
+# Note: The model and training settings do not follow the reference settings
+# from the paper. The settings are chosen such that the example can easily be
+# run on a small dataset with a single GPU.
+
+import pytorch_lightning as pl
+import torch
+import torchvision
+from torch import nn
+
+from lightly.models import utils
+from lightly.models.modules import AIMPredictionHead, MaskedCausalVisionTransformer
+from lightly.transforms import AIMTransform
+
+
+class AIM(pl.LightningModule):
+    def __init__(self) -> None:
+        super().__init__()
+
+        img_size = 224
+        self.patch_size = 32
+        self.num_patches = (img_size // self.patch_size) ** 2
+
+        vit = MaskedCausalVisionTransformer(
+            img_size=img_size,
+            patch_size=self.patch_size,
+            embed_dim=768,
+            depth=12,
+            num_heads=12,
+            qk_norm=False,
+            class_token=False,
+            no_embed_class=True,
```

Comment:
It would be better to pass the values of these parameters at the `init` of `AIM`

Reply:
I will keep them hardcoded. The examples are optimized to be short and concise, we do the same for all the other examples.